finesse/config.py

```diff
@@ -53,6 +53,25 @@
 EM27_SENSORS_TOPIC = "em27_sensors"
 """The topic name to use for EM27 sensor-related messages."""
 
+DECADES_HOST = "localhost"
+"""The IP address or hostname of the DECADES server."""
+
+DECADES_SENSORS_URL = "http://{host}/decades/livedata"
+"""The URL of the DECADES sensor data endpoint."""
+
+DECADES_SENSORS_POLL_INTERVAL = 60.0
```

Comment:
We might want a higher poll rate than this, especially when it comes to plotting the air temperature over time, but we can probably leave it as this for now.

Reply:
No worries. I'll leave it for now.